functions/src/firestore.ts

```diff
@@ -0,0 +1,194 @@
+import {AqiBufferElement, Pm25BufferElement, BufferStatus} from './buffer';
+
+const SENSORS = 'sensors';
+
+/**
+ * Interface for the structure of a sensor's data, used in the `sensors` collection.
```

Comment:
Use capitalized variable name instead of our specific collection name (e.g. `SENSORS` instead of `sensors` (I comment on this throughout the PR, lmk if you disagree)